camel/agents/chat_agent.py

```diff
@@ -314,7 +320,9 @@ def update_memory(
             MemoryRecord(message=message, role_at_backend=role)
         )
 
-    def set_output_language(self, output_language: str) -> BaseMessage:
+    def set_output_language(
+        self, output_language: str
+    ) -> Optional[BaseMessage]:
```

Comment:
Let make `You must output text in {output_language}.` as the system message instead of `None` if `output_language ` is not `None`.